frontend/shared-components/link-button/link-button.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react"
+import styles from "./link-button.module.css"
+import classnames from "classnames"
+
+export interface Props extends React.HTMLProps<HTMLButtonElement> {
+  loading?: boolean
+}
+export default function LinkButton({ className, loading, children, ...rest }: Props) {
+  return (
+    <button {...rest} className={classnames(className, styles.linkButton)} type="submit">
+      {loading ? <Spinner /> : children}
```

Comment:
I don't think we'll need a loading state for these links so why don't weremove the loading prop and Spinner